src/classNames.js

```diff
@@ -0,0 +1,15 @@
+import style from 'flexboxgrid';
+
+function defaultClassNameLookup(className) {
+  return style[className] || className;
+}
+
+let classNameLookup = defaultClassNameLookup;
+
+export function setClassNameLookup(fn) {
```

Comment:
No, it's not needed for my use case. 

I left it there so it'd provide a hook to customize styles in case someone wanted to do some sort of BEM type prefix.  I agree though, better to remove it for now and it can be added later if there's a demonstrated need.  Have pushed an update to remove